plugins/orchestrator/src/components/ExecuteWorkflowPage/ExecuteWorkflowPage.tsx

```diff
@@ -66,12 +72,23 @@ export const ExecuteWorkflowPage = (props: ExecuteWorkflowPageProps) => {
       for (const key in formState) {
         if (formState.hasOwnProperty(key)) {
           const property = formState[key];
-          Object.assign(parameters, property);
+          Object.assign(parameters, { [key]: property });
         }
       }
     }
 
     setLoading(true);
+    const workflowType =
+      schemaResponse?.workflowItem.definition.annotations?.find(
+        annotation => annotation === ASSESSMENT_WORKFLOW_TYPE,
+      )
+        ? WorkflowCategory.ASSESSMENT
+        : WorkflowCategory.INFRASTRUCTURE;
```

Comment:
We now have a function called `getWorkflowCategory` that does exactly that.
The problem is that this function is currently placed in the backend plugin (plugins/orchestrator-backend/src/helpers/workflows.ts).
Could you please move the function to the `orchestrator-common` package and reuse it here?
IMO we can move the function to plugins/orchestrator-common/src/workflow.ts.
I don't want to block this PR so if you prefer we can do it in a separate PR.

Reply:
Done. Please review

Reply:
TYVM!!